Find a functionally equivalent code:com.android.internal.telephony.CallerInfoAsyncQuery.allocate:COMMENT
Method Modifier: private     internal    
Comment:/**
 * Method to create a new CallerInfoAsyncQueryHandler object, ensuring correct
 * state of context and uri.
 */

@@ -3,8 +3,7 @@
         throw new QueryPoolException("Bad context or query uri.");
     }
     mHandler = new CallerInfoAsyncQueryHandler(context);
-    mHandler.mQueryContext = context;
     mHandler.mQueryUri = contactRef;
 }

